spec/models/manageiq/providers/openstack/infra_manager/refresher_rhos_juno_spec.rb

```diff
@@ -7,7 +7,7 @@
     @ems = FactoryGirl.create(:ems_openstack_infra, :zone => zone, :hostname => "192.0.2.1",
                               :ipaddress => "192.0.2.1", :port => 5000)
     @ems.update_authentication(
-        :default => {:userid => "admin", :password => "3182d1c220549bdfdb50777666cdb1a2caa8472b"})
+      :default => {:userid => "admin", :password => "1e4b5875bdd1a857e91d92d40f7082e607691986"})
```

Comment:
@ladas, thought for another PR:  Can you have the environment builder set the password to something static?  Then, this isn't changing each time you test with a newly built environment.